src/datalad_installer.py

```diff
@@ -87,6 +94,7 @@ def parse_log_level(level: str) -> int:
         return lv
 
 
+@dataclass
```

Comment:
wondering for self-education: what is the motivation/need for using `@dataclass` for these classes now? The concise "Add --help-versions option to miniconda component" commit message gives no such information, making me wondering either it was just a generic refactoring to improve code "while at it" or needed to accomplish the drill ?

Reply:
It was just a generic refactoring.